### PR TITLE
Pull request with pre-commit hooks and files corrected for it. 

### PR DESCRIPTION
**Pull request by aneeshn**
Please read up here : 
https://pre-commit.com/
https://github.com/PyCQA/isort

### CODE REVIEW

**Review by aliceyang16 — COMMENTED** [review]
I disapprove of these changes

**Comment by aneeshn** [comment]
> I disapprove of these changes

Please comment in code where you disaprove

File: timer.py [hunk 1]
```diff
@@ -31,7 +30,3 @@ def startTimer(self, t):
 			time.sleep(1)
 			t -= 1
 		print("ALL DONE")
```
Comment (aliceyang16) [inline timer.py]: Where's my code, dude?!

Comment (aliceyang16) [reply timer.py]: What is this?!

Comment (aneeshn) [reply timer.py]: Unnecessary code deleted

**Pull request merged**